src/plugins/dam/components/asset-screen/TranscodeableCard.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { Badge, Card, CardBody, CardHeader, Table } from 'reactstrap';
+import { Link } from 'react-router-dom';
+import artifactUrl from 'plugins/ovp/artifactUrl';
+import AssetId from '@triniti/schemas/triniti/dam/AssetId';
+import camelCase from 'lodash/camelCase';
+import damUrl from 'plugins/dam/damUrl';
+import NodeRef from '@gdbots/pbj/well-known/NodeRef';
+import nodeUrl from 'plugins/ncr/nodeUrl';
+import startCase from 'lodash/startCase';
+import withPbj from 'components/with-pbj';
+
+function TranscodeableCard({ asset, pbj }) {
+  const status = asset.has('transcoding_status') ? asset.get('transcoding_status').getValue() : 'unknown';
+  const videoId = AssetId.fromString(NodeRef.fromNode(asset).getId());
+  const imageId = AssetId.fromString(`image_jpg_${videoId.getDate()}_${videoId.getUuid()}`);
+
+  const image = pbj !== null ? pbj.set('_id', imageId) : null;
```

Comment:
@meghedi this just makes a dummy node to be used for other purposes? interesting

Reply:
@efjacobson yes, I figured this is a good way to get the image if you agree.

Reply:
yeah its a useful util, hadnt seen it before